### DISCUSSION TITLE
Does this work with Coral Dev Board?

### DISCUSSION THREAD

**Discussion by bugsysiegals** [blakeblackshear/frigate#2984] [Ask A Question] [answered]
Will Frigate work with the Dev Board?  

I'd prefer the M.2 or USB but can't find it anywhere other than people selling them for $400 on eBay.  Is there any vendor who will have some stock within the next few months?

**Comment by NickM-27** [comment, marked as answer]
I've heard of a few users using it although definitely heard of some issues and lesser performance compared to the other types. There are a few issues related to the dev board in this repo

**Reply by NickM-27** [reply]
https://github.com/blakeblackshear/frigate/issues/1010
https://github.com/blakeblackshear/frigate/issues/2852

**Reply by bugsysiegals** [reply]
Thanks!  It seems I will have to hang tight for the M.2 to come back into supply.  About this, my motherboard is Rog Strix B450-F Gaming which is supposed to have 2 M.2 slots of type M.  Any idea if it would help to buy 2 of these?  I'm not sure it can leverage parallel processing or if it's even needed as I've only 10 cameras.

**Reply by NickM-27** [reply]
> Thanks!  It seems I will have to hang tight for the M.2 to come back into supply.  About this, my motherboard is Rog Strix B450-F Gaming which is supposed to have 2 M.2 slots of type M.  Any idea if it would help to buy 2 of these?  I'm not sure it can leverage parallel processing or if it's even needed as I've only 10 cameras.

Frigate can utilize them in parallel but I don't think more than one is needed for 10 cameras for sure
